src/main/java/uk/gov/companieshouse/company_appointments/CompanyAppointmentMapper.java

```diff
@@ -118,11 +120,20 @@ private LinksView mapLinks(CompanyAppointmentData companyAppointmentData) {
     }
 
     private DateOfBirth mapDateOfBirth(CompanyAppointmentData companyAppointmentData) {
-        return Optional.ofNullable(companyAppointmentData.getData().getDateOfBirth())
-                .map(dateOfBirth -> new DateOfBirth(
-                        dateOfBirth.getMonthValue(),
-                        dateOfBirth.getYear()))
-                .orElse(null);
+        if (registerView) {
+            return Optional.ofNullable(companyAppointmentData.getData().getDateOfBirth())
+                    .map(dateOfBirth -> new DateOfBirth(
+                           dateOfBirth.getDayOfMonth(),
+                           dateOfBirth.getMonthValue(),
+                           dateOfBirth.getYear()))
+                    .orElse(null);
+        }else{
```

Comment:
This else would look better spaced out
```suggestion
        } else {
```